plugin-development-tools/src/createRollupConfig.ts

```diff
@@ -86,8 +85,8 @@ function getPlugins(
     }),
     mode === 'npm' && sourceMaps(),
     mode === 'npm' && writeIndex(packageName, distPath),
-    (mode === 'esmBundle' || mode === 'umd') && globals(),
-    (mode === 'esmBundle' || mode === 'umd') && builtins(),
+    (mode === 'esmBundle' || mode === 'umd') &&
+      nodePolyfills({ include: null }),
```

Comment:
i didn't fully understand the null option reading https://github.com/FredKSchott/rollup-plugin-polyfill-node#options what did you find it useful for? could add a comment

Reply:
I added a comment:

```js
      // By default, nodePolyfills only polyfills code in node_modules/. We set
      // to null here to include the plugin srouce code itself (and for Yarn 2/3
      // compatiblity, since it doesn't use node_moduels/).
      nodePolyfills({ include: null }),
```